crates/bevy_pbr/src/render/mesh.wgsl

```diff
@@ -1,5 +1,6 @@
-#import bevy_pbr::mesh_view_bind_group
-#import bevy_pbr::mesh_struct
+#import bevy_pbr::mesh_view_types
+#import bevy_pbr::mesh_types
```

Comment:
Should we move the mesh binding to a `mesh_bindings` import? It seems like that would also benefit from this pattern.